tests/e2e/tests/critical-path/tree-view/delimiter.e2e.ts

```diff
@@ -0,0 +1,55 @@
+import {acceptLicenseTermsAndAddDatabase, deleteDatabase} from '../../../helpers/database';
+import {
+    BrowserPage
+} from '../../../pageObjects';
+import {
+    commonUrl,
+    ossStandaloneBigConfig
+} from '../../../helpers/conf';
+import {rte} from '../../../helpers/constants';
+
+const browserPage = new BrowserPage();
+
+fixture `Delimiter tests`
+    .meta({type: 'critical_path'})
+    .page(commonUrl)
+    .beforeEach(async() => {
+        await acceptLicenseTermsAndAddDatabase(ossStandaloneBigConfig, ossStandaloneBigConfig.databaseName);
+    })
+    .afterEach(async() => {
+        await deleteDatabase(ossStandaloneBigConfig.databaseName);
+    })
+test
+    .meta({rte: rte.standalone})
```

Comment:
Could be configured in the fixture level

Reply:
Fixed